canonical-websites/www.canonical.com
# Convert /careers to Vanilla v1 styles

## Done

Upgrade /careers to v1

## QA

- Pull code
- Check markup at in all template files under `templates/careers`
- Run `./run serve --watch`
- Check pages locally at 


http://0.0.0.0:8002/careers
http://0.0.0.0:8002/careers/all-design-and-user-experience-vacancies
http://0.0.0.0:8002/careers/all-marketing-and-business-development-vacancies
http://0.0.0.0:8002/careers/all-operations-vacancies
http://0.0.0.0:8002/careers/all-professional-services-vacancies
http://0.0.0.0:8002/careers/all-technical-and-engineering-vacancies
http://0.0.0.0:8002/careers/all-vacancies
http://0.0.0.0:8002/careers/cloud-vacancies
http://0.0.0.0:8002/careers/go-vacancies

Fixes https://www.github.com/ubuntudesign/vanilla-squad/issues/30

## Demo

http://www.canonical.com-homepage-polish.demo.haus/careers
